lib/src/package_config_impl.dart

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import "package_config.dart";
+export "package_config.dart";
+import "util.dart";
+
+class SimplePackageConfig implements PackageConfig {
+  final int version;
```

Comment:
What version is this? The package config schema version? The root package's pub version? The root package's SDK version? A rolling version as the package config for a workspace get's updated with subsequent "pub get" calls?

Should this have a longer name?

Can we add `@override` annotations so that I would know to look elsewhere for a doc comment?

Reply:
This is the version defined in the `PackageConfig` interface, which is also the documentation I get when I hover the declaration in my IDE.
Not sure what longer name would be better. It would be redundant to call it `packageConfigurationVersion`.

Reply:
An override annotation would resolve the confusion without repeating a doc comment. Override annotations are standard for Google authored Dart.